src/validation_utils.py

```diff
@@ -1,5 +1,10 @@
 import pandas as pd
 
+# Colors
```

Comment:
Now this is defined twice (also in `validation.py`), remove one or the other.

Reply:
Personally, I would create a function for info/error/warning, but this solution is okay as well.